tests/js-custom-authorization/custom_authorization.py

```diff
@@ -29,9 +39,281 @@ def run(args):
         network = test_custom_auth(network, args)
 
 
+@reqs.description("Test stack size limit")
+def test_stack_size_limit(network, args):
+    primary, _ = network.find_nodes()
+
+    with primary.client("user0") as c:
+        r = c.post("/app/recursive", body={"depth": 50})
+        assert r.status_code == http.HTTPStatus.OK, r.status_code
+
+    with primary.client("user0") as c:
+        r = c.post("/app/recursive", body={"depth": 2000})
+        assert r.status_code == http.HTTPStatus.INTERNAL_SERVER_ERROR, r.status_code
+
+    return network
+
+
+@reqs.description("Test heap size limit")
+def test_heap_size_limit(network, args):
+    primary, _ = network.find_nodes()
+
+    with primary.client("user0") as c:
+        r = c.post("/app/alloc", body={"size": 5 * 1024 * 1024})
+        assert r.status_code == http.HTTPStatus.OK, r.status_code
+
+    with primary.client("user0") as c:
+        r = c.post("/app/alloc", body={"size": 500 * 1024 * 1024})
+        assert r.status_code == http.HTTPStatus.INTERNAL_SERVER_ERROR, r.status_code
+
+    return network
+
+
+def run_limits(args):
+    with infra.network.network(
+        args.nodes, args.binary_dir, args.debug_nodes, args.perf_nodes, pdb=args.pdb
+    ) as network:
+        network.start_and_join(args)
+        network = test_stack_size_limit(network, args)
+        network = test_heap_size_limit(network, args)
+
+
+@reqs.description("JWT authentication")
+def test_jwt_auth(network, args):
```

Comment:
This and other tests have moved here but isn't JS-specific I don't think?

Reply:
No, they're not, they're just a grouping of things that deal with auth/validation and can run on single node network, so they can be run at the same time without timeouts. I've deliberately not renamed anything, but we probably will want to. Input welcome!

Reply:
We can keep the files split, the reason I moved them is because they weren't usable/importable as modules where they were.